test/Modal.spec.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
 import { shallow, mount } from 'enzyme';
 import Modal from '../src/Modal';
-import mocker from './helper/mocker';
+import mocker from './helper/new-mocker';
 
 describe('<Modal />', () => {
   let wrapper;
-  const modalMock = jest.fn();
-  const restore = mocker('modal', modalMock);
+  const modalInitMock = jest.fn();
+  const modalInstanceDestroyMock = jest.fn();
+  const modalMock = {
+    init: (el, options) => {
```

Comment:
you would need to add here `open` method mock
```js
const modalOpenMock = jest.fn();
const modalMock = {
  open: modalOpenMock
  ...
```

Reply:
Hey @alextrastero, I've done that, but It still throws me the same error. Please, check the last commit